src/components/transport_manager/test/CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ set(EXCLUDE_PATHS
 
```

Comment:
I suppose platform_specific directory should be added depending on the target platform.
Other should be excluded.

Reply:
@EKuliiev fixed